common/static/common/templates/components/paging-header.underscore

```diff
@@ -1,17 +1,17 @@
 <% if (!_.isUndefined(srInfo)) { %>
-  <h2 class="sr" id="<%= srInfo.id %>"><%- srInfo.text %></h2>
+  <h2 class="sr" id="<%= HtmlUtils.ensureHtml(srInfo.id) %>"><%- srInfo.text %></h2>
 <% } %>
 <div class="search-tools listing-tools">
     <span class="search-count listing-count">
-        <%= message %>
+        <%= HtmlUtils.ensureHtml(message) %>
```

Comment:
Can this message be HTML? If not, I work use:

    <%- message %>

If it can, our convention is to name the variable with a suffix to make it clear, i.e. ``messageHtml``.